src/plugins/data/server/search/strategies/esql_async_search/esql_async_search_strategy.ts

```diff
@@ -46,12 +54,12 @@ export const esqlAsyncSearchStrategyProvider = (
   }
 
   function asyncSearch(
-    { id, ...request }: IKibanaSearchRequest<SqlQueryRequest['body']>,
+    { id, ...request }: IKibanaSearchRequest<ESQLQueryRequest['body']>,
```

Comment:
Since we are using `dropNullColumns` from request params, can we revert changes to `body` type?

Reply:
Sorry Julia, I am not following

Reply:
It seems to be redundant to add `dropNullColumns` also as a part of `body` type when it's already added to `ESQLSearchParams` type.

```suggestion
    { id, ...request }: IKibanaSearchRequest<SqlQueryRequest['body']>,
```

Reply:
Synced with Julia offline and we agreed on https://github.com/elastic/kibana/pull/176236/commits/4a448e8465c41eca0a00ed08655a08450095809f